packages/utils/src/collections.ts

```diff
@@ -18,6 +18,16 @@ export function zip3<A, B, C>(as: A[], bs: B[], cs: C[]) {
   return res
 }
 
+export function compact<A>(as: Array<A | null>): A[] {
```

Comment:
Seems like this could be done with `.filter`?

Reply:
Great point, originally filter didn't properly get rid of the type, but i can use the custom defined type guard below